Dockerfile

```diff
@@ -4,17 +4,13 @@ LABEL "maintainer"="Sviatoslav Sydorenko <wk+re-actors@sydorenko.org.ua>"
 LABEL "repository"="https://github.com/re-actors/gh-action-pypi-publish"
 LABEL "homepage"="https://github.com/re-actors/gh-action-pypi-publish"
 
-LABEL "com.github.actions.name"="pypi-publish"
-LABEL "com.github.actions.description"="Upload Python distribution packages to PyPI"
-LABEL "com.github.actions.icon"="upload-cloud"
-LABEL "com.github.actions.color"="yellow"
-
 ENV PYTHONDONTWRITEBYTECODE 1
 ENV PYTHONUNBUFFERED 1
 
-ADD LICENSE.md /LICENSE.md
```

Comment:
I think I'd like to ship the license in the image...

Reply:
I'm by no means familiar with any legal stuff, but to me it seems odd to put the LICENSE.md which applies for the Dockerfile and other contents of the repo into the rootfs of the image, which contains a lot of things licensed under other licenses and you've no ownership of.

Reply:
Fair enough. So maybe put it under `/app`?